packages/client-ui/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@dharpa-vre/client-ui",
+  "version": "0.1.0",
+  "description": "DHARPA toy VRE",
+  "keywords": [
+    "dharpa",
+    "vre",
+    "toy"
+  ],
+  "homepage": "https://dharpa.org",
+  "main": "src/index.ts",
+  "license": "AGPL-3.0-only",
+  "author": "roman@kalyakin.com",
+  "scripts": {
+    "watch": "tsc -w",
+    "eslint": "eslint src --ext .ts,.tsx,.js,.jsx --fix",
+    "eslint:check": "eslint src --ext .ts,.tsx,.js,.jsx",
+    "build:lib": "tsc"
+  },
+  "peerDependencies": {
+    "@dharpa-vre/client-core": "*",
+    "react": "17"
+  },
+  "devDependencies": {
+    "@types/node": "14",
+    "@types/react": "17",
+    "@types/react-dom": "17",
+    "@types/react-router-dom": "^5.1.7",
+    "@typescript-eslint/eslint-plugin": "4",
+    "@typescript-eslint/parser": "4",
+    "eslint": "7",
+    "eslint-config-prettier": "7",
+    "eslint-plugin-prettier": "3",
+    "eslint-plugin-react": "7",
+    "prettier": "2",
+    "typescript": "4"
+  },
+  "dependencies": {
+    "@fontsource/roboto": "^4.2.2",
+    "@material-ui/core": "^4.11.3",
```

Comment:
I guess `@material-ui` dependencies are here just to start with. Very soon we will want the same dependencies in `modules` package and in some new packages (for example some reusable rich components). So we may need to extract that into a `styles` package (or some other similar name). That's also where we can add theme tweaks.

Reply:
Sounds like a great idea, though I have to admit I don't have a clear picture of how that would look. I suppose we can get to this later, though, can't we?